app/code/Magento/Ui/view/base/web/js/core/renderer/layout.js

```diff
@@ -600,6 +602,25 @@ define([
             });
 
             return result;
+        },
+
+        /**
+         * Get sort order for child if it is undefined.
+         *
+         * @param {Object} parent
+         * @param {Object} child
+         */
+        getChildSortOrderIfUndefined: function (parent, child) {
```

Comment:
This method does not return anything, but naming suggests so (`get`).

Reply:
Fixed.